src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-serviceclients.js

```diff
@@ -26,6 +26,227 @@
 
 module.exports = {
   tags: ['ss', 'clients', 'serviceclients'],
+  'Security server service clients check services': (
+    browser,
+  ) => {
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+    const clientsTab = mainPage.section.clientsTab;
+    const clientInfo = mainPage.section.clientInfo;
+    const serviceClientsPage = browser.page.serviceClients.serviceClientsPage();
+    const serviceClientDetails = browser.page.serviceClients.serviceClientDetails();
+
+    // Open SUT and check that page is loaded
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage.signinDefaultUser();
+
+    // Navigate to service client 
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+    clientsTab.openClient('TestClient');
+    browser.waitForElementVisible(clientInfo);
+    browser.expect.element(clientInfo.elements.serviceClientsTab).to.be.visible;
+    clientInfo.openServiceClientsTab();
+    browser.waitForElementVisible(serviceClientsPage.section.serviceClientsTab);
+    serviceClientsPage.openServiceClient('TestCom');
```

Comment:
This code block seems to be in use in all three tests in this file. Should create a function for it.

Reply:
Refactored as suggested